app/views/docs/realtime.phtml

```diff
@@ -23,7 +23,7 @@ $channels = [
 
 <p>Instead of requesting new data via HTTP, the subscription will receive new data every time it changes, any connected client receives that update within milliseconds via a WebSocket connection.</p>
 
-<p>This lets you build an interactive and responsive user experience by providing information from all of Appwrite's services in realtime.</p>
+<p>This lets you build an interactive and responsive user experience by providing information from all of Appwrite's services in realtime. The example below shows subscribing to realtime file changes.</p>
```

Comment:
```suggestion
<p>This lets you build an interactive and responsive user experience by providing information from all of Appwrite's services in realtime. The example below shows subscribing to realtime file creation.</p>
```

Let's be specific, as code has this `if` only showing create events (not update, not delete)